### DISCUSSION TITLE
select2 livewire validation error message not showing wire:ignore

### DISCUSSION THREAD

**Discussion by RVP04** [livewire/livewire#8149] [1. Help]
I have blade file which contains the code 
```
<div class="col-md-5 mb-3" wire:ignore>
                    <label class="form-label" for="branches">{{__('Branch Name')}} <span
                            class="text-danger"> *</span> </label>
                    <select data-placeholder="{{__('Branch Name')}}..." name="branch"
                            required
                            id="branches"
                            class="select2-branch form-control @error('branch') is-invalid @enderror">
                        <option value="">{{__('Branch Name')}}.</option>
                    </select>
                    @if($errors->has('branch'))
                        <div class="invalid-feedback">
                            <strong>{{ $errors->first('branch') }}</strong>
                        </div>
                    @else
                        <div class="invalid-feedback">
                            {{__('Please choose a branch')}}
                        </div>
                    @endif
                </div>
```
if I delete ``` wire:ignore``` it works fine. 

if I include then error validation not showing anything is possible because theme supports only when added inside the div. 

I tried placing into the select2 element it disappears.

**Comment by danie-ramdhani** [comment]
> `wire:ignore` will instruct Livewire to ignore the contents of a particular element, even if they change between requests.
> https://livewire.laravel.com/docs/wire-ignore

if you have `live` condition, you need to place it outside `wire:ignore` or use `wire:ignore.self` to ignore specific element.

**Reply by RVP04** [reply]
Problem 1: 
If I use wire:ignore.self in select the entire html element disappears 

Problem 2:
The boot strap theme I have choose required that invalid-feedback inside the div which makes error message as hidden 

Any alternative solution please let me know @danie-ramdhani

**Reply by danie-ramdhani** [reply]
```blade
<div wire:ignore.self>
...
           <select data-placeholder="{{__('Branch Name')}}..." name="branch"
                            required
                            id="branches"
                            wire:ignore.self // HERE
                            class="select2-branch form-control @error('branch') is-invalid @enderror"> // why using `@error` here? it not gonna work
                        <option value="">{{__('Branch Name')}}.</option>
                    </select>
                    ..
</div>
```

i've using select2 in the past but remove it because it using jQuery for the deps.
if you're need ui for livewire, maybe you can try maryui (https://mary-ui.com/docs/components/choices#select-all), there's similar select2 too, but using tailwindcss not bootstrap.

**Reply by RVP04** [reply]
@danie-ramdhani i don't whether it will conflict or not will check thank you bro
